### PR TITLE
Jinja2 capabilities for more fields in pushover plugin

### PR DESCRIPTION
**Pull request by liiight**
Added ability to use jinja2 with 'sound' and 'priority' values in Pushover plugin

### CODE REVIEW

File: flexget/plugins/output/pushover.py [hunk 1]
```diff
@@ -101,6 +105,21 @@ def on_task_output(self, task, config):
                 log.warning("Problem rendering 'urltitle': %s" % e)
                 urltitle = ""
 
+            # Attempt to render the priority field
+            if isinstance(priority, basestring):
+                try:
+                    priority = entry.render(priority)
+                except ValueError as e:
+                    log.warning('Problem rendering "priority": %s' % e)
+                    priority = 0
```
Comment (gazpachoking) [inline flexget/plugins/output/pushover.py]: Is it a problem that priority might be a string or an int after this?

Comment (liiight) [inline flexget/plugins/output/pushover.py]: Priority should be sent as an int to the api. If it's not its ignored and set to 0 by default at pushover side so there's no harm.

Comment (gazpachoking) [inline flexget/plugins/output/pushover.py]: entry.render will always return a string, should it be `priority = int(entry.render(priority))` then?

Comment (liiight) [inline flexget/plugins/output/pushover.py]: I'm sorry, I was mistaken. Priority CAN be string, it's resolved on API side. I actually tested exactly this and it worked.

**Pull request merged**